### PR TITLE
Add Minus Games

### PR DESCRIPTION
**Pull request by Accessory**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/minus-games.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+source_code_url: "https://github.com/Accessory/minus_games"
+description: "Minus Games is a server and launcher combo that helps you sync game and save files across multiple devices."
+licenses:
+  - MIT
+platforms:
+  - Rust
+  - Docker
+tags:
+  - Games - Administrative Utilities & Control Panels
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/minus-games.yml:13]: Optional so can be removed,
```suggestion
```

File: software/minus-games.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# software name
+name: "Minus Games"
+website_url: "https://github.com/Accessory/minus_games"
+source_code_url: "https://github.com/Accessory/minus_games"
+description: "Minus Games is a server and launcher combo that helps you sync game and save files across multiple devices."
```
Comment (Ki-er) [inline software/minus-games.yml:5]:
```suggestion
description: "Server and launcher combo that helps you sync game and save files across multiple devices."
```

File: software/minus-games.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+licenses:
+  - MIT
+platforms:
+  - Rust
+  - Docker
```
Comment (Ki-er) [inline software/minus-games.yml:10]: > https://accessory.github.io/minus_games_user_guide/quick_start/quick_start.html

Docker not present.

```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes

**Review by kokomo123 — APPROVED** [review]
LGTM, No issues here. Thanks for the corrections, and thanks for your contribution!

File: software/minus-games.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# software name
+name: "Minus Games"
+website_url: "https://github.com/Accessory/minus_games"
+source_code_url: "https://github.com/Accessory/minus_games"
+description: "Server and launcher combo that helps you sync game and save files across multiple devices."
```
Comment (nodiscc) [inline software/minus-games.yml:5]: I don't understand what the software does, and the README doesn't help much. Can you point me to a doc explaining what the features are, what the projects' goals/primary use cases are, etc?

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/minus-games.yml (hunk 4 above)
Comment (Accessory) [reply software/minus-games.yml:5]: More infos and a Guide can be found @ https://accessory.github.io/minus_games_user_guide/

File: software/minus-games.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# software name
+name: "Minus Games"
+website_url: "https://accessory.github.io/minus_games_user_guide"
+source_code_url: "https://github.com/Accessory/minus_games"
+description: "Server and launcher combo that helps you sync game and save files across multiple devices."
```
Comment (nodiscc) [inline software/minus-games.yml:5]:
```suggestion
description: "Sync games and save files across multiple devices."
```
Cleaner description, in my opinion

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/minus-games.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# software name
```
Comment (Rabenherz112) [inline software/minus-games.yml:1]: Let's also remove the comment
```suggestion
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**